linera-chain/src/chain.rs

```diff
@@ -780,7 +780,8 @@ where
         replaying_oracle_responses: Option<Vec<Vec<OracleResponse>>>,
     ) -> Result<BlockExecutionOutcome, ChainError> {
         #[cfg(with_metrics)]
-        let _execution_latency = metrics::BLOCK_EXECUTION_LATENCY.measure_latency();
+        let _execution_latency =
+            metrics::BLOCK_EXECUTION_LATENCY.measure_latency(MeasurementUnit::Milliseconds);
```

Comment:
If message and operation execution is in microseconds, I'd make block execution microseconds, too?